core/src/main/java/com/radixdlt/logger/EventLoggerModule.java

```diff
@@ -219,12 +221,13 @@ private static void processLedgerUpdate(
     */
   }
 
-  private static void logEpochChange(BFTValidatorId self, EpochChange epochChange) {
+  private static void logEpochChange(SelfValidatorInfo self, EpochChange epochChange) {
     var validatorSet = epochChange.getBFTConfiguration().getValidatorSet();
+    final var included = self.bftValidatorId().stream().anyMatch(validatorSet::containsValidator);
```

Comment:
I dislike the readability of calling `anyMatch` on an Optional, but I guess this is the best in-built Java way of doing this?

Reply:
I'm afraid so :man_shrugging: . There's good ol' if-else but that's arguably uglier than anyMatch